packages/editor/src/managers/clipboard/copy-cut-manager.ts

```diff
@@ -172,6 +172,7 @@ export class CopyCutManager {
         e.preventDefault();
         e.stopPropagation();
       } catch (e) {
+        // FIXME: throws `DOMException: Modifications are not allowed for this document` in Firefox
```

Comment:
if (isFireFox) { 
  ignore
} else {
  throw
}

Reply:
I have no idea why it doesn't work in Firefox. 🤣